lang/lang_en_fr.txt

```diff
@@ -42,6 +50,10 @@
 "Ambient"
 "Ambiant"
 
+#MSG_AUTO c=6
+"Auto"
+"\x00"
+
 #MSG_PRESS c=20
 "and press the knob"
 "et pressez le bouton"
```

Comment:
This translation translates to "and squeeze the button." I believe that a better translation would be `et poussez le bouton`

Reply:
@awenelo found other "press the knob" and it has been traslated to `et appuyez sur le bouton` Is that okay?

Reply:
Yes, `appuyez sur` would work for "press".
A few other translations that I noticed:

`lit` translates to "bed"(as in the bed you sleep on) or is the 3rd person present tense of the verb "to read" (he/she reads)

- If it is supposed to be "heated bed", `plateau chauffant`(heating plate) would be a better option
- If it is supposed to be "Please read", `lis`(2nd person imperative) would be a better conjugation.

I see three different translations of "Steel sheets":
- `Feuilles d'acier`, which is "Steel Sheets" (as in paper sheets) or "Steel leaves" (as in tree leaves)
- `Plaque en acier`, which is "Steel Plates"
- `Plaque d'impression`, which is "Printing Plates"

`Ventilo tete` translates to "head fan" (I read this to be more a fan for your head). As there are no other fans that I am aware of, we can probably change it to `Ventilo`(familiar version of fan) or `ventilateur`(more formal version of fan)

When reading `Mesure de la hauteur de reference du point de calibration`, I take it as a command to Measure the reference height of the calibration point. Another way that this could be understood is that it is displaying, somewhere with that text, the reference height of the calibration point. Also,  in french, `calibration` is the age of a historical object. A better translation would probably be `Je mesure la hauteur de reference de la point de calibrage `

Also, `Calibration`, when a translation of the verb to calibrate should be some conjugation of the verb `Calibrer`. If its a command ("calibrate") it should be translated to `calibre`, and if it's "I'm calibrating", `Je calibre`.

Reply:
When reading Mesure de la hauteur de reference du point de calibration, I take it as a command to Measure the reference height of the calibration point. Another way that this could be understood is that it is displaying, somewhere with that text, the reference height of the calibration point. Also, in french, calibration is the age of a historical object. A better translation would probably be Je mesure la hauteur de reference de la point de calibrage 

"je mesure la hauteur de reference du point de calibrage"

Reply:
> et appuyez sur le bouton

Used this.
I will try to implement the other comments you added here as well. Makes it bit more difficult as have to search for these and not commented at the translation.

Reply:
Updated all `bed` to `plateau chauffant`

Reply:
Updated `Steel Sheets` to `Plaques en acier`

Reply:
```
#MSG_SELFTEST_EXTRUDER_FAN c=20
"Left hotend fan?"
"Ventilo gauche?"
```
Hope that is clear

Reply:
```
#MSG_MEASURE_BED_REFERENCE_HEIGHT_LINE1 c=60
"Measuring reference height of calibration point"
"Je mesure la hauteur de reference de la point de calibrage"
```